src/pytezos/cli/cli.py

```diff
@@ -173,5 +185,78 @@ def deploy(
             logger.info(status)
 
 
+@cli.command(help='Run containerized sandbox')
+@click.option('--sandbox-image', type=str, help='Sandbox Docker image to use', default=SANDBOX_IMAGE)
+@click.option('--protocol', type=click.Choice(['Florence', 'Edo']), help='Protocol to use', default='Florence')
+@click.option('--port', '-p', type=int, help='Port to map container port to', default=8732)
+@click.option('--block-time', '-bt', type=float, help='Interval between calls to bake_block (in seconds)', default=1.0)
+@click.option('--n-blocks', '-n', type=int, help='Number of blocks to bake')
+@click.pass_context
+def sandbox(
+    _ctx,
+    sandbox_image: str,
+    protocol: str,
+    port: int,
+    block_time: float,
+    n_blocks: int,
+):
+    client = get_docker_client()
+    try:
+        client.images.get(sandbox_image)
+    except docker.errors.ImageNotFound:
+        logger.info('Will now pull latest sandbox image, please stay put.')
+        image, tag = sandbox_image.split(':')
+        for line in client.api.pull(image, tag=tag, stream=True, decode=True):
+            logger.info(line)
+        logger.info('Pulled sandbox image successfully!')
+
+    logger.info('Starting node.')
+    node = client.containers.run(
+        sandbox_image,
+        ports={
+            '8732': ('localhost', port)
+        },
+        detach=True
+    )
+
+    atexit.register(node.stop)
+
+    logger.info('Giving node 5 seconds to start.')
+
+    def end_waiting():
+        logger.info('Waiting over.')
+
+    waiting_for_node_start_thread = threading.Timer(5, end_waiting)
```

Comment:
5 seconds is too much, better poll /version endpoint as it's done in sandbox testcase